bloomfilters/matt-keibler/src/BloomFilter.cpp

```diff
@@ -0,0 +1,29 @@
+#include "BloomFilter.h"
+
+BloomFilter::BloomFilter(HashCalculator* hash_calculator_, vector<string> initial_words) : hash_calculator(hash_calculator_) {
+  bloom = new bitset<UINT_MAX + 1ul>();
```

Comment:
m of max unsigned int + 1 as an unsigned long int ([limits.h reference](http://www.cplusplus.com/reference/climits/)). I derived m and k from the hash function. The hash returns an unsigned long int, which I then split up into sub-hashes to use for the algorithm. I first tried an m of max unsigned short int + 1 and a k of 4, but that ended up with too many positive bits and so it failed to identify words as misspelled.

I'll move them to constants that show their relation, but I don't think it would be easy to change these without changing the hash algorithm as well.